homeassistant/components/unifiprotect/config_flow.py

```diff
@@ -97,10 +97,17 @@ async def _async_get_nvr_data(
             errors[CONF_PASSWORD] = "invalid_auth"
         except NvrError as ex:
             _LOGGER.debug(ex)
-            errors["base"] = "nvr_error"
+            errors["base"] = "unknown"
```

Comment:
If it's a known exception we normally don't label it as unknown. What does this error mean?

Reply:
`NvrError` is out base class exception for the python library. So, it is a generic error. The exact cause may be know (401 authenticated for example), but we do not have errors in the python library built out for all of the possible ones. But basically, this would be "check the logs" type error.

We had discussed internally just use the one already provided by HA instead having our own `nvr_error` message for consistency.

Reply:
I think this is OK for now.  I can't think of any other common exceptions that we could actually classify that we don't already do (the authentication failure).

Reply:
If it's a request failure, isn't it more a case of `cannot_connect`?

Reply:
That one probably does make more sense. Switched it over.

Reply:
Yeah that is much better 👍  👍  👍